server/metadata.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"fmt"
+)
+
+// jobMetadata holds information we save in post props
+// to persist some state needed by call initiated jobs
+// (e.g. recordings, transcriptions).
+type jobMetadata struct {
+	// FileID is the FileInfo.Id that the job produced (e.g. recording file).
+	FileID string
+	// PostID is the Post.Id that holds the job's artifacts (e.g. recording post).
+	PostID string
+	// RecID is the recording ID.
+	RecID string
+	// TrID is the transcription ID.
+	TrID string
```

Comment:
I think we lose some clarity if we do that because in the recording job's metadata we store the job id for the related transcription and vice versa. If we simply call it `JobID` I feel it gets even more confusing. Plus we already omit empty fields so there's isn't a huge concern.

Reply:
That makes sense.